### DISCUSSION TITLE
What Next..

### DISCUSSION THREAD

**Discussion by crazynightgriffin** [visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA#470] [General]
So the hack has been installed for just under 24hrs and my mind is already spinning with possibilities of turning the pump off setting different temperatures depending on current temperature etc but fighting the urge and keeping it simple at the moment.

At the moment I have it set to heat to 40'c between 00:30 and 04:30 then 38'c every other time with the pump running 24/7

How have you setup your spa once the unit is installed and why?

<img width="1020" alt="image" src="https://user-images.githubusercontent.com/66025008/225322580-28305517-ab65-4707-88c9-a51b3fe8c527.png">

**Comment by wyliecoyoteuk** [comment]
I heat mine to 40C between 00:30 and 04:30AM. Then turn I it off. I turn the pump back on @ 8:00AM.
My wife uses it for 30 minutes exercise around 10:00AM most days.
It is pretty well insulated and in a summerhouse, so most days it is still at 38-39C around 4:00PM when we tend to use it together.
I have it set set that if it is below 38C @03:30PM, it comes on and heats to 39C, which is our preferred temperature.
One thing to note is that the tub will only display 40C, yet it may get up to 42C before it turns off.

**Reply by visualapproach** [reply]
- [ ] When we had our spa I set it to 38 over the weekends when we usually used it, and then only ran the filter a couple of ours per day the rest of the week. It saved the impeller lifespan I imagined. I went through a few. If I'd have it now when I'm on hourly tariff I'd heat it during the cheapest hours just as I do with some other stuff. All automated with HA and market prices for today and tomorrow available online.
Another cool thing I did was to open and close the awnings above the spa, by pressing on the timer button :)

**Reply by visualapproach** [reply]
I would probably also make it so a button press on the display would send a message to my wife's phone asking for a beer 😄
